pylint/testutils/output_line.py

```diff
@@ -62,39 +76,42 @@ def __init__(self, row, exception):
 class OutputLine(NamedTuple):
     symbol: str
     lineno: int
-    column: str
-    object: Any
+    column: int
+    object: str
     msg: str
-    confidence: interfaces.Confidence
+    confidence: str
 
     @classmethod
-    def from_msg(cls, msg):
+    def from_msg(cls, msg: Message) -> "OutputLine":
         column = cls.get_column(msg.column)
         return cls(
             msg.symbol,
             msg.line,
             column,
             msg.obj or "",
             msg.msg.replace("\r\n", "\n"),
-            msg.confidence.name
-            if msg.confidence != interfaces.UNDEFINED
-            else interfaces.HIGH.name,
+            msg.confidence.name if msg.confidence != UNDEFINED else HIGH.name,
         )
 
     @classmethod
-    def get_column(cls, column):
-        if not PY38_PLUS:
-            return ""  # pragma: no cover
-        return str(column)
+    def get_column(cls, column: str) -> int:
+        if (
+            not PY38_PLUS
+        ):  # This mimicks behaviour of MessagesHandlerMixIn.add_one_message()
```

Comment:
This is referring to the fact that for `not PY38_PLUS` the `column` is always `0`. Perhaps this must be made clearer, or put in a docstring. Let me know what your preferences are.

Reply:
I think not taking the column into account is only for tests though ? Because below python 3.8 the ast information are different (and inferior) so we're checking only above python 3.8.

Reply:
I think you are right! My comment was referring to perhaps making the comment a little bit clearer, although I don't immediately know how.

Reply:
```suggestion
        ):  # We check the column only for the new better parser introduced in python 3.8
```

Reply:
```suggestion
        ):  # We check the column only for the new better ast parser introduced in python 3.8
```
😄